packages/dds/tree/api-report/tree.api.md

```diff
@@ -1342,7 +1342,7 @@ export enum NodeKind {
 }
 
 // @internal
-export type NoListenersCallback<E extends Events<E>> = (eventName: keyof Events<E>) => void;
+export type NoListenersCallback<TListeners extends object> = (eventName: keyof Listeners<TListeners>) => void;
```

Comment:
Was the type change here intentional?

Reply:
Just noticed the note in the PR description. Seems like the answer is yes :)

Reply:
Yes :) In a couple places we have to do `TListeners extends Listeners<TListeners>` in order for TS to typecheck correctly, so we were doing it conservatively everywhere. I checked each one and simplified the ones we could.